apps/api/src/domain/models/bibliographic-reference/shared/bibliographic-reference-data-union.ts

```diff
@@ -0,0 +1,5 @@
+import { Union } from '@coscrad/data-types';
+import { BIBLIOGRAPHIC_REFERENCE_DATA_UNION } from './bibliographic-reference-union-data-member.decorator';
+
+@Union(BIBLIOGRAPHIC_REFERENCE_DATA_UNION, 'type')
```

Comment:
Is there any documentation that needs to be updated?

Reply:
Let's do that as part of [this story](https://www.pivotaltracker.com/story/show/185987878).